googletest/include/gtest/internal/gtest-port.h

```diff
@@ -393,8 +393,11 @@ typedef struct _RTL_CRITICAL_SECTION GTEST_CRITICAL_SECTION;
 #if GTEST_HAS_ABSL
 // When using Abseil, RE2 is required.
```

Comment:
I am quite unsure about the logic behind this statement.
It looks like it should be reversed, i. e.

> When using RE2, Abseil is required.